### ISSUE TITLE
Poison Extract Misty World not spawning

### ISSUE THREAD

**Issue by KingofThemyscira** [davqvist/Exoria#263]
This is not a bug but it's an issue.  You need poison extract from misty world.  The problem is they're nearly impossible to come by.  I have about 30 of these sponge trees I'm watching as they cycle through growth stages.  In hours of camping these things I've only ever seen 1 produce a poison extract.  

Either I'm doing something wrong here or I need some help getting these things.

**Comment by davqvist** [comment]
In my testing there were plenty of Sponges around in the Misty World Down Biome and most of them had Poison Extract and when you right click them to harvest they easily grew back. The question is if my testing is the outlier or your case. I recommend exploring a bit in the Down Biome and look if you find more grown up versions. Also don't mind me asking: You right click the orange blob at the top right and not cut down the whole sponge?

**Comment by KingofThemyscira** [comment]
yes I right click the top when the orange bulb is there.  I'm in the down forest biome, I planted a ton of them and there are even more naturally spawning.  Juts not getting any of the poison forming, I've been working on this since yesterday mid day.  I even took 20 seeds back to my base and they haven't produced any either.

**Comment by davqvist** [comment]
Hm, weird, I will do some testing.

**Comment by KingofThemyscira** [comment]
@davqvist have you got a solution on this issue, I've done pretty much everything I can up to this point but this.  Been stuck playing PoE for 2 days since I can't progress.

**Comment by davqvist** [comment]
I have not done the testing sorry, but I have talked to some beta testers. They don't have that issues, it takes some time, sure, but they don't have issues like you have. So I am not sure.

**Comment by davqvist** [comment]
You are the only one having this issue so far, I can't replicate it in any way or form. Sorry
